src/codecs/music_xmp.c

```diff
@@ -89,6 +97,13 @@ static int XMP_Load(void)
         FUNCTION_LOADER(xmp_stop_module, void(*)(xmp_context))
         FUNCTION_LOADER(xmp_release_module, void(*)(xmp_context))
         FUNCTION_LOADER(xmp_free_context, void(*)(xmp_context))
+#if defined(XMP_DYNAMIC)
+        libxmp.xmp_load_module_from_callbacks = (int (*)(xmp_context,void*,struct xmp_callbacks)) SDL_LoadFunction(libxmp.handle, "xmp_load_module_from_callbacks");
```

Comment:
Reasonable to copy my `FUNCTION_LOADER_OPTIONAL` macro I made at my side (used for tempo code only, but also can be used here too):
```c
#ifdef XMP_DYNAMIC
#define FUNCTION_LOADER(FUNC, SIG) \
    libxmp.FUNC = (SIG) SDL_LoadFunction(libxmp.handle, #FUNC); \
    if (libxmp.FUNC == NULL) { SDL_UnloadObject(libxmp.handle); return -1; }
#define FUNCTION_LOADER_OPTIONAL(FUNC, SIG) \
    libxmp.FUNC = (SIG) SDL_LoadFunction(libxmp.handle, #FUNC);
#else
#define FUNCTION_LOADER(FUNC, SIG) \
    libxmp.FUNC = FUNC;
#define FUNCTION_LOADER_OPTIONAL(FUNC, SIG) \
    libxmp.FUNC = FUNC;
#endif
```

Reply:
The next code can be simplified then:
```c
#if (XMP_VERCODE >= 0x040500) || defined(XMP_DYNAMIC)
        FUNCTION_LOADER_OPTIONAL(xmp_load_module_from_callbacks, int (*)(xmp_context,void*,struct xmp_callbacks));
#else
        libxmp.xmp_load_module_from_callbacks = NULL;
#endif
```